app/player/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.player.players import Player
+
+
+class Dwarf(Player):
```

Comment:
Indicate that `Dwarf` is also an abstract class inheriting it from the `ABC` since it does not implement all abstract methods from the `Player` class.